http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/app/CromwellRunnerAppInstall.scala

```diff
@@ -79,6 +84,26 @@ class CromwellRunnerAppInstall[F[_]](config: CromwellRunnerAppConfig,
         AppCreationException(s"Pet not found for user ${params.app.auditInfo.creator}", Some(ctx.traceId))
       )
 
+      maybeProfile <- stringToUUID(params.billingProfileId.value) match {
+        case Success(uuid) => bpmClient.getProfile(userToken, uuid)
+        case _             => F.pure(None)
+      }
+
+      maybeLimits <- maybeProfile match {
+        case Some(billingProfile) =>
+          Option(billingProfile.getOrganization.getLimits) match {
+            case Some(limits) if !limits.isEmpty =>
+              val scalaLimits = limits.asScala
+              val value = scalaLimits.get("maxconcurrentworkflows")
+              value match {
+                case Some(v) => F.pure(Some(raw"config.maxConcurrentWorkflows=${v}"))
+                case None    => F.pure(None)
+              }
+            case _ => F.pure(None)
+          }
+        case _ => F.pure(None)
+      }
```

Comment:
you can use flatMap/map to unnest all the various options in this block. I think this can simplify to:
```scala
      maybeLimits = maybeProfile.flatMap(
        _.getOrganization.getLimits.asScala
          .get("maxconcurrentworkflows")
          .map(v => raw"config.maxConcurrentWorkflows=${v}")
      )
```

note my solution is slightly different - you are handling a case where `billingProfile.getOrganization.getLimits` returns null. Can this actually return null?

For `Option`s, `.map` means "if the current value is Some, do the next block of code, else return None" and the leading `flatMap` means "don't return Option[Option[Option[something]]], flatten them all out"